dramatiq/rate_limits/backends/memcached.py

```diff
@@ -100,6 +100,8 @@ def incr_and_sum(self, key, keys, amount, maximum, ttl):
                 if value > maximum:
                     return False
 
+                # TODO: Drop non-callable keys in Dramatiq v2.
+                keys = keys() if callable(keys) else keys
```

Comment:
This should be renamed to `key_list` or something since it's in a loop to avoid shadowing the `keys` callable.